test/e2e/common/node/runtimeclass.go

```diff
@@ -51,7 +51,21 @@ var _ = SIGDescribe("RuntimeClass", func() {
 		handler := f.Namespace.Name + "-handler"
 		rcName := createRuntimeClass(f, "unconfigured-handler", handler)
 		pod := f.PodClient().Create(e2enode.NewRuntimeClassPod(rcName))
-		expectSandboxFailureEvent(f, pod, handler)
+		eventSelector := fields.Set{
+			"involvedObject.kind":      "Pod",
+			"involvedObject.name":      pod.Name,
+			"involvedObject.namespace": f.Namespace.Name,
+			"reason":                   events.FailedCreatePodSandBox,
+		}.AsSelector().String()
+		// Events are unreliable, don't depend on the event. It's used only to speed up the test.
+		err := e2eevents.WaitTimeoutForEvent(f.ClientSet, f.Namespace.Name, eventSelector, handler, framework.PodEventTimeout)
+		if err != nil {
+			framework.Logf("Warning: did not get event about FailedCreatePodSandBox")
```

Comment:
WaitTimeoutForEvent could return an `err` if the timeout happens or unexpected error.
It would be nice to output `err` in the `Logf` message.

Reply:
Good point, I included err in the log statement.